src/file.cpp

```diff
@@ -197,7 +197,7 @@ namespace
 		for (int i = 0; i < num_bufs; ++i)
 		{
 			DWORD num_read;
-			if (ReadFile(fd, bufs[i].iov_base, bufs[i].iov_len, &num_read, &ol[i]) == FALSE
+			if (ReadFile(fd, bufs[i].iov_base, DWORD(bufs[i].iov_len), &num_read, &ol[i]) == FALSE
```

Comment:
do you think it would be possible to define the iovec_t member iov_len as a DWORD? maybe it would save some of these casts